libretro/sdl-libretro/SDL_config.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _SDL_config_h
```

Comment:
if this port requires SDL 1.2, is it a good idea to #error out if the SDL version being used is a higher versin?